Formula/bowtie2.rb

```diff
@@ -15,12 +15,14 @@ class Bowtie2 < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "46f6e5aa64fd0f9564d15f21aa0ac84d9960fb79424539599c284d0f7724e096"
   end
 
-  depends_on "simde"
-  depends_on "tbb"
```

Comment:
Default TBB was replaced by C++ threads in 2.4.3: https://github.com/BenLangmead/bowtie2/releases/tag/v2.4.3